### ISSUE TITLE
About Loss Function and Preprocessing

### ISSUE THREAD

**Issue by chankl3579** [CorentinJ/Real-Time-Voice-Cloning#908]
Hi,

Could you please explain how is the loss calculated?
I read from you codes that L1 L2 loss are used, but how is it actually calculated, especially when ground truth and predicted spectrogram are not in the same shape? The same question to the cross entropy loss.
May I also ask what are the preprocessing you do to the training data?

Many thanks.
